icons/folder_type_private.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>folder_type_private</title><path d="M27.5,5.5H18.2L16.1,9.7H4.4V26.5H29.6V5.5Zm0,4.2H19.3l1.1-2.1h7.1Z" style="fill:#c15368"/><path d="M20.4,10A10.5,10.5,0,1,0,30.9,20.5,10.5,10.5,0,0,0,20.4,10Zm0,18.375a7.862,7.862,0,0,1-6.41-12.432L24.958,26.911A7.826,7.826,0,0,1,20.4,28.375Zm6.411-3.318L15.843,14.09A7.862,7.862,0,0,1,26.811,25.057Z" style="fill:#d80027"/></svg>
```

Comment:
I'd change the folder color to #913E4E to improve the contrast with the forbidden icon
![image](https://cloud.githubusercontent.com/assets/696981/26755925/d12f7e4e-4897-11e7-8d95-45966ade4ab0.png)